recipes/sapiens/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
I've opened a PR upstream to add the license file to the source distribution. If/when that's merged and there is a new upstream release of `sapiens`, this manually added copy should be removed.

Reply:
Merged and released 1.0.1, shall we change it here already? 

The new sha256 is `49eedaec6213410423ec3b46e2655fbf8731caee7de69fa2873a59497274bae7`

Reply:
If you want you can update the version and sha256 here and delete the manually packaged license file